src/client/components/PasswordForm.tsx

```diff
@@ -228,7 +237,7 @@ export const PasswordForm = ({ submitUrl, fieldErrors }: Props) => {
 
   return (
     <form
-      css={form}
+      css={[form, gridAutoRow && gridAutoRow(passwordFormSpanDef)]}
```

Comment:
if this password form is on a page which uses a css grid layout, like the consents flow, calling `gridAutoRow` sets up the css grid correctly for this form element

Reply:
nice 👍